CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 Based on the suggestion at https://keepachangelog.com/en/1.0.0/.
 
 ## [Unreleased]
+- Added support for the Elk reasoner (as "elk") and made it the default reasoner.
```

Comment:
I suppose this isn't wrong. But it seems there's another important change in how a phylorefs are modeled that goes along with this change (or in fact makes it possible to start with, no?). So doesn't this leave something out that's at least as worthy to report?

Reply:
True! I've added it now.